src/seeds/seeds.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require_relative '../library'
+
+lib = Library.new('../storage.yml')
+
+author = Author.new('Nazar')
+author2 = Author.new('Trololo', 'Norm')
+lib.add(author)
+lib.add(author2)
+
+book = Book.new('Book1', author)
+book2 = Book.new('Book2', author2)
+lib.add(book)
+lib.add(book2)
```

Comment:
Propose to separate the seed and the main app.

Reply:
done